### PR TITLE
Start Basic Plan trial on Registration

### PR DESCRIPTION
**Pull request by alex-deaconu**
Only for new Users (not users added to existing Company)

Conditional wording for Sign Up Page

Inform user of Trial starting
if creating a new Company

User added to Existing Company wording

Improved warning visibility for Google Account related warnings

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@Rise-Vision/apps please review. Thanks!

**Comment by alex-deaconu** [comment]
Staged at https://apps-stage-5.risevision.com

File: test/e2e/cases/plans.js [hunk 1]
```diff
@@ -28,33 +28,34 @@
       });
 
       describe("Plan banner", function () {
-        it("should show free plan banner", function () {
-          expect(homepage.getFreePlanBanner().isDisplayed()).to.eventually.be.true;
+        it("should show Basic Plan Trial", function () {
```
Comment (ezequielc) [inline test/e2e/cases/plans.js]: @alex-deaconu I am not sure I follow the changes in e2e test here. When we sign in, we sign in with Jenkins account, and it shouldn't be on Basic Plan Trial, it should continue as Free. Maybe I am missing something. Thanks

Comment (alex-deaconu) [reply test/e2e/cases/plans.js]: @ezequielc CH e2e tests always register and remove the jenkins1@risevision.com account. So, this account is registered as a new User and the Trial is started automatically as per the new flow. Thanks.

File: src/templates/add-user-email.html [hunk 2]
```diff
@@ -32,7 +32,7 @@
                       display: inline-block;
                       padding: 10px;
                       background-color: #f5f5f5; text-align:center;">
-                                          <p><span>Visit our <a href="https://apps.risevision.com/signup" target="_blank">Sign Up</a> page and use the email address</span>
+                                          <p><span>Visit our <a href="https://apps.risevision.com/joinaccount/{{newUser.encodedCompanyName}}" target="_blank">Sign Up</a> page and use the email address</span>
```
[the remark this answers is not in the document]
Comment (alex-deaconu) [reply src/templates/add-user-email.html]: @ezequielc good catch, updated. Thanks!

**Pull request merged**